requirements/edx/github.txt

```diff
@@ -9,9 +9,9 @@
 
 # Third-party:
 -e git+https://github.com/cyberdelia/django-pipeline.git@1.5.3#egg=django-pipeline
-git+https://github.com/edx/django-wiki.git@ned/dj18#egg=django-wiki==1.0
--e git+https://github.com/edx/django-oauth2-provider.git@will/django-1.8-upgrade#egg=django-oauth2-provider==1.0
-git+https://github.com/edx/django-openid-auth.git@django1.8-upgrade#egg=django-openid-auth==1.0
+git+https://github.com/edx/django-wiki.git@ned/dj18#egg=django-wiki==0.0.4
```

Comment:
Sorry again, never mind, we didn't change it.  "master" has no relation to what we are running.  Upstream changed the name, we've always used "django-wiki".  Sorry for the turmoil.